packages/@webex/plugin-cc/src/constants.ts

```diff
@@ -1,4 +1,9 @@
 export const POST_AUTH = 'postauth';
 export const WCC_API_GATEWAY = 'wcc-api-gateway';
 export const EVENT = 'event';
+export const READY = 'ready';
 export const SUBSCRIBE_API = 'v1/notification/subscribe';
+export const REGISTER_TIMEOUT = 20000;
```

Comment:
The query regarding this timeout value in my existing comment, how we came up with this number is not answered